packages/components/accordion/src/accordion.stories.ts

```diff
@@ -227,6 +229,78 @@ export const Sticky: Story = {
   }
 };
 
+export const ToggleExternally: Story = {
+  render: ({ single }) => {
+    try {
+      customElements.define(
+        'accordion-toggle-example',
+        class extends LitElement {
+          alien = false;
+          dino = true;
+
+          override render(): TemplateResult {
+            return html`
+              <sl-button @click=${this.toggleDino}>Toggle 🦖</sl-button>
+              <sl-button @click=${this.toggleAlien}>Toggle 👽</sl-button>
+              <p>
+                🦖 state: ${this.dino ? 'open' : 'closed'}<br />
+                👽 state: ${this.alien ? 'open' : 'closed'}
+              </p>
+              <sl-accordion ?single=${single}>
```

Comment:
It looks like a `single` attribute is not being applied to the accordion, even when it's set.